package.json

```diff
@@ -27,10 +27,10 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "eslint": "^5.16.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-react": "^7.12.4"
+    "eslint-plugin-react": "^7.12.4",
+    "fsevents": "^1.2.7"
```

Comment:
@MandelaK, what does this dependency do?

Reply:
Tests aren't running on some machines without it ... There are two ways to fix it, either install watchman globally using brew and also install xcode, or you can install fsevents as a dev dependency ... I read about this fix here https://github.com/expo/expo/issues/854#issuecomment-343243804 and informed the team about the fix in both @BrianSerem's PR and in the slack channel